pages/docs/integrations/grouparoo-intercom.mdx

```diff
@@ -11,132 +11,195 @@ Grouparoo's [Intercom](https://www.intercom.com/) Plugin enables you to export c
 
 This guide will show you how to work with the Intercom Plugin to [create a Destination](#create-an-intercom-destination) to export your data.
```

Comment:
It looks like the "Create an Intercom Destination" heading was removed below, this link doesn't take me anywhere.

Reply:
Thanks, good catch. I've removed this.